app/build.gradle

```diff
@@ -72,6 +72,9 @@ dependencies {
     compile "com.android.support:design:${SUPPORT_LIB_VERSION}"
     compile 'com.squareup.picasso:picasso:2.5.2'
     compile 'com.squareup:otto:1.3.8'
+    compile('com.github.afollestad.material-dialogs:core:0.8.5.3@aar') {
```

Comment:
Yes this provide many additional functionality like changing visual effects
of the dialogue box, specially custom views and other effects which are
very easy to implement with this library.
https://github.com/afollestad/material-dialogs


Sent with MailTrack
<https://mailtrack.io/install?source=signature&lang=en&referral=vishwajeets912@gmail.com&idSignature=22>



Sent with MailTrack
<https://mailtrack.io/install?source=signature&lang=en&referral=vishwajeets912@gmail.com&idSignature=22>

*Vishwajeet Srivastava*
B.Tech, Pre-Final (Computer Science and Engineering)
The LNM Institute of Information Technology
Mob:+91-9461787316

On Mon, Jan 25, 2016 at 12:08 AM, Manan Wason <notifications@github.com>
wrote:

> In app/build.gradle
> <https://github.com/fossasia/open-event-android/pull/170#discussion_r50639784>
> :
>
> > @@ -72,6 +72,9 @@ dependencies {
> >      compile "com.android.support:design:${SUPPORT_LIB_VERSION}"
> >      compile 'com.squareup.picasso:picasso:2.5.2'
> >      compile 'com.squareup:otto:1.3.8'
> > +    compile('com.github.afollestad.material-dialogs:core:0.8.5.3@aar') {
>
> Why are you using this? You can use the one from AppCompat. Does this
> offer anything extra?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fossasia/open-event-android/pull/170/files#r50639784>.
>